src/openloco/windows/industry_list.cpp

```diff
@@ -684,7 +684,7 @@ namespace openloco::ui::windows::industry_list
 
         static int getRowIndex(int16_t x, int16_t y)
         {
-            return (x / 122) + (y / rowHeight) * 5;
+            return (x / 112) + (y / rowHeight) * 5;
```

Comment:
It would be best to file these smaller fixes in a separate PR to get them merged earlier.

Reply:
Sure I can do that.